doofinder-for-woocommerce/doofinder-for-woocommerce.php

```diff
@@ -162,6 +162,58 @@ public function __construct()
             }
         }
 
+        public static function filter_woocommerce_prices_with_taxes( $price, $product ) {
```

Comment:
Very good job, just one question. Is this file the best place to have functions related to the API?

Wouldn't it be better to leave this file clean of functions that are not basic and have these that execute business plan in a separate class from the part we call API and call it directly from initialize_rest_endpoints ?

Just IMO, great work!

Reply:
Thank you very much for the feedback. I think you are right, so I'll try to change it :smile:

Reply:
Changed on: https://github.com/doofinder/doofinder-woocommerce/pull/172/commits/44044d3f62e9afd5dd4771e29bcafe2757ece4ca